providers/shared/layers/Environment/id.ftl

```diff
@@ -142,6 +142,12 @@
         {
             "Names" : "FlowLogs",
             "Children" : [
+                {
+                    "Names" : "Enabled",
+                    "Description" : "Don't use - here to override automatically added enabled attribute",
+                    "Types"  : BOOLEAN_TYPE,
+                    "Default" : false
+                },
```

Comment:
Might be better to say Deprecated instead of Don't use

Reply:
done